test/acceptance/features/steps/steps.py

```diff
@@ -190,6 +206,78 @@ def then_sbo_jq_is(context, jq_expression, sbr_name, json_value_regex):
 
 
 # STEP
+is_serverless_operator_running_step = u'Openshift Serverless Operator is running'
+
+
+@given(is_serverless_operator_running_step)
+def given_serverless_operator_is_running(context):
+    """
+    Checks if the serverless operator is up and running
+    """
+    serverless_operator = ServerlessOperator()
+    if not serverless_operator.is_running():
+        print("Serverless operator is not installed, installing...")
+        serverless_operator.install_operator_subscription() | should.be_truthy.desc(
+            "serverless operator subscription installed")
+        serverless_operator.is_running(wait=True) | should.be_truthy.desc("serverless operator installed")
+    print("Serverless operator is running!!!")
+    context.serverless_operator = serverless_operator
+
+
+# STEP
+is_quarkus_s2i_image_present_step = u'Quarkus s2i builder image is present'
+
+
+@given(is_quarkus_s2i_image_present_step)
+def given_quarkus_builder_image_is_present(context):
+    """
+    Checks if quarkus s2i builder image is present
+    """
+    builder_image = BuilderImage()
+    if not builder_image.is_present():
+        builder_image.import_and_patch() | should.be_truthy.desc("Quarkus image is imported from image registry and patched to builder")
```

Comment:
it looks to me that the assertion error messages are not correct, for example here: the error gets triggered if `import_and_patch` returns false, but then the message should be "Quarkus image is iNOT mported from image registry and patched to builder"

Same goes for the next message. Please double check other messages in this PR.

Reply:
I have replaced should with assert with corresponding error message.